app/coffeescripts/models/Assignment.coffee

```diff
@@ -29,6 +30,9 @@ define [
       if (turnitin_settings = @get('turnitin_settings'))?
         @set 'turnitin_settings', new TurnitinSettings(turnitin_settings),
           silent: true
+      if (vericite_settings = @get('vericite_settings'))?
+        @set 'vericite_settings', new VeriCiteSettings(vericite_settings),
```

Comment:
There is only one field required in the database for VeriCite:

alter table assignments ADD vericite_enabled boolean

"vericite_settings" is a logical function that uses the "turnitin_settings" database field. This allows VeriCite and Turnitin to have different advanced settings while still using the same database field.